lib/hammer_cli/defaults.rb

```diff
@@ -0,0 +1,139 @@
+require 'hammer_cli'
+require 'yaml'
+module HammerCLI
+
+  class DefaultsCommand < HammerCLI::AbstractCommand
+
+    class ListDefaultsCommand < HammerCLI::DefaultsCommand
+      command_name 'list'
+
+      desc _('List all the defaults parameters')
+      def execute
+        list_all_defaults_message
```

Comment:
It only prints plain yaml to output. Would you mind to add some more formatting? I imagine the output as:
```
hammer defaults list
organization_id:  (provided by foreman)
location_id:      1
login:            admin
```
or something similar. Instead of current:
```
hammer defaults list
organization_id:
---
:from_server: true
:provider: 0
```
Btw I'm not sure why the provider is `0` here.